### PR TITLE
Handle MS update

### PR DESCRIPTION
**Pull request by stulees**
@tejohnso @rodrigopavezi The integration test I added is forcing the message to be broadcast. On a future PR I'd like to change this to be triggered. 

FYI the test task in CCI workflow is failing because there's no termination after tests complete. Thoughts on how to fix that? Same happens locally, but if you run them individually (test-unit, test-integration) they terminate fine.

### CODE REVIEW

File: src/messaging/update/update.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    const {files} = message;
+    const filesForUpdate = entry.validateAndFilter(files);
+
+    if (!filesForUpdate) {
+      return Promise.reject(new Error("Invalid update message"));
+    }
+
+    return Promise.all(filesForUpdate.map((file) => {
+      return Promise.all([db.fileMetadata.put, db.watchlist.put].map((action) => {
+        const dbEntry = Object.assign({}, file, {status: "STALE"});
```
Comment (tejohnso) [inline src/messaging/update/update.js]: @stulees The LS Watchlist shouldn't have a status field

Comment (stulees) [reply src/messaging/update/update.js]: @tejohnso To keep the code condensed, I opted to not do that as the db watchlist update will ignore fields other than `filePath` and `version`. Do you prefer to be more explicit and not do this?

Comment (tejohnso) [reply src/messaging/update/update.js]: @stulees can you clarify where the ignore is occurring?

Comment (stulees) [reply src/messaging/update/update.js]: @tejohnso See here - https://goo.gl/ku5g8Q

Comment (tejohnso) [reply src/messaging/update/update.js]: I had looked at that but for some reason I imagined it would still add all fields ...somehow.

**Comment by stulees** [comment]
@tejohnso See last commit and https://boneskull.com/mocha-v4-nears-release/#otherbreakingchanges . I can't figure out from our tests what is not being cleaned up so I opted for the --exit flag to force exit like it used to do.

**Comment by tejohnso** [comment]
Ok, perhaps I'll run it locally later and give it a shot.

**Pull request merged**